### PR TITLE
[canary] M1 canary test

### PR DESCRIPTION
**Pull request by xyang16**

### CODE REVIEW

**Review by zachgk — APPROVED** [review]

File: .github/workflows/canary-arm64-osx.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          distribution: 'corretto'
+          java-version: 11
+          architecture: aarch64
+      - name: Test PyTorch
+        working-directory: canary
+        run: |
+          set -x
+          DJL_ENGINE=pytorch-native-auto PYTORCH_VERSION=$PT_VERSION ./gradlew clean run
+          rm -rf /root/.djl.ai/
+          DJL_ENGINE=pytorch-native-auto PYTORCH_VERSION=1.10.0 ./gradlew clean run
```
Comment (frankfliu) [inline .github/workflows/canary-arm64-osx.yml:44]: I don't think we support 1.10.0 for M1

File: .github/workflows/canary-arm64-osx.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        with:
+          distribution: 'corretto'
+          java-version: 11
+          architecture: aarch64
+      - name: Test PyTorch
+        working-directory: canary
+        run: |
+          set -x
+          DJL_ENGINE=pytorch-native-auto PYTORCH_VERSION=$PT_VERSION ./gradlew clean run
+          rm -rf /root/.djl.ai/
```
Comment (frankfliu) [inline .github/workflows/canary-arm64-osx.yml:43]: Is this folder correct for M1?

Comment (xyang16) [reply .github/workflows/canary-arm64-osx.yml]: Fixed.

File: .github/workflows/canary-arm64-osx.yml (hunk 1 above)
Comment (xyang16) [reply .github/workflows/canary-arm64-osx.yml]: Fixed.

**Pull request merged**